src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working.");

```

Comment:
The message in the `doWork` method should be more informative. It should indicate that the 'Excavator' is working, not just started working. For example, 'Excavator is excavating.'